kostyor/conf.py

```diff
@@ -0,0 +1,13 @@
+from oslo_config import cfg
+config_files = ['conf.ini']
```

Comment:
Agree, let's put the defaults in this file and then have `rest_api.py` have an optional argument to the path. Flask already has an environment variable that it uses for configuration. We may be able to utilize.

http://flask.pocoo.org/docs/0.11/config/